src/observability/kubearmor.go

```diff
@@ -179,11 +179,11 @@ func fetchSysServerConnDetail(log types.KubeArmorLog) (types.SysObsNwData, error
 	if strings.Contains(log.Data, "tcp_connect") || strings.Contains(log.Data, "SYS_CONNECT") {
 		conn.InOut = "OUT"
 		conn.Count++
-		conn.UpdatedTime = time.UnixDate
+		conn.UpdatedTime = time.Now().Format(time.UnixDate)
 	} else if strings.Contains(log.Data, "tcp_accept") || strings.Contains(log.Data, "SYS_ACCEPT") {
 		conn.InOut = "IN"
 		conn.Count++
-		conn.UpdatedTime = time.UnixDate
+		conn.UpdatedTime = time.Now().Format(time.UnixDate)
```

Comment:
I think this time is updated when summary is invoked. Instead, use log.UpdatedTime and convert the same to this format.

Reply:
Kindly verify the changes.